core/src/main/java/nl/nn/adapterframework/doc/model/FrankDocModel.java

```diff
@@ -572,7 +581,12 @@ void buildGroups() {
 			log.warn(String.format("Name \"[%s]\" cannot been used for others group because it is the name of an ElementType", OTHER));
 		}
 		else {
-			groupsBase.put(OTHER, Arrays.asList(FrankDocGroup.getInstanceFromFrankElements(OTHER, membersOfOther)));
+			Collections.sort(membersOfOther);
```

Comment:
Ik zou het prettiger vinden als de elementen in de geparsede volgorden zouden blijven staan, dus eerst Configuration, dan Adapter, dan Receiver etc.

Reply:
Ik heb even geprobeerd wat er gebeurt als ik deze sort weghaal. Er komt dan niet eerst Configuration, dan Adapter, etc. Er komt een volgorde uit die ik niet begrijp. Wil je nu nog steeds dat ik de sort weghaal?

Reply:
Je zou eventueel een uur kunnen besteden aan begrijpen hoe de volgorde tot stand komt en bedenken of het eenvoudig is die volgorde te veranderen.

Reply:
Dat is gelukt. We sorteren de members van groep Other nu ongeveer op hun afstand tot de begin class Configuration. De sort die hier stond is nu weg.